book/src/example.md

```diff
@@ -1,493 +1,12 @@
 # Example
 
-> ## *This page is out-of-date and under active development.*
+The [Hostpath CSI driver](https://github.com/kubernetes-csi/csi-driver-host-path) is a simple sample driver that provisions a directory on the host. It can be used as an example to get started writing a driver, however it is not meant for production use.
+The [deployment example](https://github.com/kubernetes-csi/csi-driver-host-path#deployment) shows how to deploy and use that driver in Kubernetes.
 
-The
-[HostPath](https://github.com/kubernetes-csi/drivers/tree/master/pkg/hostpath)
-can be used to provision local storage in a single node test. This
-section shows how to deploy and use that driver in Kubernetes.
+The example deployment uses the original RBAC rule files that are maintained together with sidecar apps and deploys into the default namespace. A real production should copy the RBAC files and customize them as explained in the comments of those files.
 
-The deployment of a CSI driver determines which RBAC rules are
-needed. For example, enabling or disabling leadership election changes
-which permissions the external-attacher and external-provisioner need.
-This example deployment uses the original RBAC rule files that are
-maintained together with those sidecar apps and deploys into the
-default namespace.
+The example deployment includes livenessprobe side-container provided by the CSI community, using configuration derived from [using-livenessprobe](https://github.com/kubernetes-csi/livenessprobe#using-livenessprobe).
```

Comment:
Do we need to explicitly call out liveness probe and snapshot?  We're not calling out any of the other sidecars.

Reply:
current structure is result of mechanically replacing one chapter with one-two sentences about it. Other sidecars did not have dedicated chapters.

Reply:
let's leave them out then.  The new docs page has separate pages for each of the sidecars that explain what they do.

Reply:
we leave out livenessprobe and snapshot sentences, and we keep RBAC chapter?

Reply:
yup I think that is fine

Reply:
pushed it. The remaining text is not long :-)